plugin/audit_log/buffer.c

```diff
@@ -189,6 +189,7 @@ int audit_log_buffer_write(audit_log_buffer_t *log, const char *buf, size_t len)
       log->write_func(log->write_func_data, buf, len, LOG_RECORD_COMPLETE);
       audit_log_buffer_resume(log);
     }
+    audit_buffer_size_too_small++;
```

Comment:
Must be accessed in a thread safe way: for 5.6 and 5.7, look up my_atomic.h, for 8.0, std::atomic